packages/icons/resources/icons/icons.json

```
@@ -3718,5 +3718,17 @@
         "iconName": "small-square",
         "tags": "square",
         "group": "miscellaneous"
+    },
+    {
+        "displayName": "Ship",
+        "iconName": "ship",
+        "tags": "transportation, ship, boat, cruise, yacht, move, supply, supply chain, trip, shipping",
+        "group": "interface"
+    },
+    {
+        "displayName": "Cargo Ship",
+        "iconName": "ship-cargo",
```

let's just call this "cargo-ship" and make the API more intuitive... will need to rename the SVG files too

```suggestion
        "iconName": "cargo-ship",
```